### PR TITLE
Modifications to iCubGenova09 files after video

### PR DESCRIPTION
**Pull request by S-Dafarra**

### CODE REVIEW

**Comment by S-Dafarra** [comment]
For some reason, the history seems too long, but the actual modifications are few.

File: iCubGenova09/hardware/mechanicals/right_leg-eb12-j4_5-mec.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
         <param name="Encoder">          182.044              182.044       </param>
         <param name="fullscalePWM">    32000               32000           </param>
         <param name="ampsToSensor">     1000.0              1000.0         </param>
-        <param name="Gearbox_M2J">       100.0               100.0         </param>
+        <param name="Gearbox_M2J">       100.0               160.0         </param>
```
Comment (pattacini) [inline iCubGenova09/hardware/mechanicals/right_leg-eb12-j4_5-mec.xml:14]: Already in `devel`?!
See https://github.com/robotology/robots-configuration/blob/devel/iCubGenova09/hardware/mechanicals/right_leg-eb12-j4_5-mec.xml#L14

File: iCubGenova09/hardware/motorControl/right_leg-eb11-j2_3-mc_service.xml [hunk 2]
```diff
@@ -47,7 +47,7 @@
                     <param name="type">                 roie            roie       </param>
                     <param name="port">                 CAN1:1:0        CAN1:2:0   </param>
                     <param name="position">             atmotor         atmotor    </param>
-                    <param name="resolution">           14400           3600      </param>
+                    <param name="resolution">           14400           14400      </param>
```
Comment (pattacini) [inline iCubGenova09/hardware/motorControl/right_leg-eb11-j2_3-mc_service.xml:50]: Already in `devel`?!
See https://github.com/robotology/robots-configuration/blob/devel/iCubGenova09/hardware/motorControl/right_leg-eb11-j2_3-mc_service.xml#L50.

**Review by pattacini — CHANGES_REQUESTED** [review]
This PR looks quite weird as some of the proposed modifications are already in the base branch (see my inline comments).

Rebasing on `robotology/robots-configuration@devel` might help shed a light on this.

**Comment by S-Dafarra** [comment]
> This PR looks quite weird as some of the proposed modifications are already in the base branch (see my inline comments).
> 
> Rebasing on `robotology/robots-configuration@devel` might help shed a light on this.

I tried to merge. I would like to avoid rebasing since this branch is widely used in the machines of the setup. The commits did not reduce, but at least those modifications disappeared.

> Also, keeping the formatting unchanged would help the review.

Sorry about that, we edited the formatting of the calibrators files in order to use a tool we recently developed: https://github.com/dic-iit/bipedal-locomotion-framework/pull/332. This tool automatically sets the calibration deltas, but it needs to prettify the ``xml`` first. The changes in the formatting have been introduced with https://github.com/robotology/robots-configuration/pull/276/commits/6503f091c0e9f38f77982ed0903d11ad3027951b. Only the deltas should have been modified.

**Review by pattacini — APPROVED** [review]
Ok great 👍🏻

**Comment by S-Dafarra** [comment]
Thanks!

**Comment by pattacini** [comment]
@S-Dafarra in the future, it might be convenient if you merge the upstream devel branch into yours before opening up the PR as I noticed these weirdnesses only by chance.

**Comment by S-Dafarra** [comment]
> @S-Dafarra in the future, it might be convenient if you merge the upstream devel branch into yours before opening up the PR as I noticed these weirdnesses only by chance.

I agree. I just merged again ``devel`` into that branch, but it keeps considering >50 commits when opening a PR even though there are no modifications. Any idea to level them up without rebasing?

**Comment by pattacini** [comment]
>Any idea to level them up without rebasing?

Given that the two branches are now identical, why don't you get it freshly from upstream?
Make a backup copy of that branch and then create a new branch in your fork straight from upstream.

You could then compare them again within your fork and finally switch to the latest one.

**Comment by pattacini** [comment]
> this branch is widely used in the machines of the setup

rebasing shouldn't interfere if you use `git pull --rebase`.

**Comment by S-Dafarra** [comment]
Apparently, this turned out to be a GitHub design choice. See https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch

In particular, notice the difference between
https://github.com/robotology/robots-configuration/compare/devel...dic-iit:devel_iCubGenova09?expand=1
and
https://github.com/robotology/robots-configuration/compare/devel..dic-iit:devel_iCubGenova09?expand=1

(there is a ``.``difference between the two addresses)

**Comment by pattacini** [comment]
Good catch 👍🏻 
I like the [`Edit` workaround](https://stackoverflow.com/a/46782679/3956237).

**Comment by S-Dafarra** [comment]
> Good catch 👍🏻
> I like the [`Edit` workaround](https://stackoverflow.com/a/46782679/3956237).

I tried, but that does not seem to work. Actually, the address with only two dots I linked about only checks the files, the commits are still screwed up. Despite having everything merged, it seems that rebasing is the only way to realign them for Github. That's disappointing

**Pull request merged**